src/Essentials/src/FileSystem/FileSystem.shared.cs

```diff
@@ -19,6 +19,9 @@ public static string AppDataDirectory
 		public static Task<Stream> OpenAppPackageFileAsync(string filename)
 			=> PlatformOpenAppPackageFileAsync(filename);
 
+		public static Task<bool> AppPackageFileExistsAsync(string filename)
+			=> PlatformAppPackageFileExistsAsync(filename);
```

Comment:
I added this API in rev 1 of this PR, but probably not needed. But, might be useful still.